src/librustdoc/html/markdown.rs

```diff
@@ -752,9 +753,13 @@ impl LangString {
                     data.ignore = Ignore::All;
                     seen_rust_tags = !seen_other_tags;
                 }
-                x if x.starts_with("ignore-") => {
+                x if {
+                    suffix = x.strip_prefix("ignore-");
+                    suffix.is_some()
+                } =>
+                {
                     if enable_per_target_ignores {
-                        ignores.push(x.trim_start_matches("ignore-").to_owned());
+                        ignores.push(suffix.unwrap().to_owned());
```

Comment:
This is a behavior change (though probably not in practice); `ignore-ignore-` would get stripped previously.